ovos_persona_server/ollama.py

```
@@ -0,0 +1,435 @@
+                done=True,
+                total_duration=total_duration,
+                load_duration=load_duration,  # Placeholder
+                prompt_eval_count=prompt_eval_count,  # Placeholder
+                prompt_eval_duration=prompt_eval_duration,  # Placeholder
+                eval_count=eval_count,  # Placeholder
+                eval_duration=eval_duration,  # Placeholder
+                done_reason=done_reason
+            ).model_dump(exclude_unset=True))
+        except Exception as e:
+            raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                                detail=f"Persona generation failed: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)